tests/foreman/api/test_lifecycleenvironment.py

```diff
@@ -24,204 +24,203 @@
+    """
+    with pytest.raises(HTTPError):
+        entities.LifecycleEnvironment(name=name).create()
 
-        :expectedresults: Lifecycle environment is created and updated properly
 
-        :CaseLevel: Integration
+@pytest.mark.parametrize('new_name', **parametrized(valid_data_list()))
+@tier1
+def test_positive_update_name(module_lce, new_name):
+    """Create lifecycle environment providing the initial name, then
+    update its name to another valid name.
 
-        :CaseImportance: Low
-        """
-        lc_env = entities.LifecycleEnvironment(
-            organization=self.org, description=gen_string('alpha')
-        ).create()
-        new_description = gen_string('utf8')
-        lc_env = entities.LifecycleEnvironment(id=lc_env.id, description=new_description).update(
-            ['description']
-        )
-        self.assertEqual(lc_env.description, new_description)
+    :id: b6715e02-f15e-4ab8-8b13-18a3619fee9e
 
-    @tier1
-    def test_negative_update_name(self):
-        """Update lifecycle environment providing an invalid name
+    :expectedresults: Lifecycle environment is created and updated properly
 
-        :id: 55723382-9d98-43c8-85fb-df4702ca7478
+    """
+    module_lce.name = new_name
+    module_lce.update(['name'])
+    updated = entities.LifecycleEnvironment(id=module_lce.id).read()
+    assert new_name == updated.name
 
-        :expectedresults: Lifecycle environment is not updated and
-            corresponding error is raised
 
-        :CaseImportance: Low
-        """
-        name = gen_string('alpha')
-        lc_env = entities.LifecycleEnvironment(organization=self.org, name=name).create()
-        for new_name in invalid_values_list():
-            with self.subTest(new_name):
-                with self.assertRaises(HTTPError):
-                    entities.LifecycleEnvironment(id=lc_env.id, name=new_name).update(['name'])
-                    self.assertEqual(lc_env.read().name, name)
+@pytest.mark.parametrize('new_desc', **parametrized(valid_data_list()))
+@tier2
+def test_positive_update_description(module_lce, new_desc):
+    """Create lifecycle environment providing the initial
+    description, then update its description to another one.
 
-    @tier1
-    @upgrade
-    def test_positive_delete(self):
-        """Create lifecycle environment and then delete it.
+    :id: e946b1fc-f79f-4e57-9d4a-3181a276222b
 
-        :id: cd5a97ca-c1e8-41c7-8d6b-f908916b24e1
+    :expectedresults: Lifecycle environment is created and updated properly
 
-        :expectedresults: Lifecycle environment is deleted successfully
+    :CaseLevel: Integration
 
-        :CaseImportance: Critical
-        """
-        lc_env = entities.LifecycleEnvironment(organization=self.org).create()
-        lc_env.delete()
-        with self.assertRaises(HTTPError):
-            lc_env.read()
+    :CaseImportance: Low
+    """
+    module_lce = entities.LifecycleEnvironment(description=gen_string('alpha')).create()
```

Comment:
@latran is it required to create new lce? In `test_positive_update_name` test you didn't created one.

Reply:
Agreed with @jameerpathan111 , you already have `lce` fixture that creates new lce for each test (that uses it).

Reply:
The `lce` fixture has no description.  I created `module_lce` to create a description and then update it.

Reply:
@latran I'm leaving that up to you, but from my point of view, it makes sense for `lce` fixture to create lce with all params, so update tests can focus on actual updates.

Reply:
Added description to fixture.